.ci/install_runtime.sh

```diff
@@ -12,6 +12,8 @@ set -o errtrace
 
 cidir=$(dirname "$0")
 
+bash "${cidir}/install_rust.sh" && source "$HOME/.cargo/env"
```

Comment:
I would call the rust install script from `build_install_agent_ctl()`.  Here it is loosely in between variables declarations...